backend/climate_match/serializers/question_answer.py

```diff
@@ -34,34 +36,39 @@ class Meta:
 
     def get_text(self, obj: Question) -> str:
         user_language_code = get_language()
+        assert obj.language
         if obj.language.language_code != user_language_code:
             return (
                 obj.translate_question.filter(
                     language__language_code=user_language_code
                 )
                 .first()
-                .text
+                .text  # type: ignore
             )
         return obj.text
 
     def get_answer_type(self, obj: Question) -> str:
+        assert obj.answer_type
         return obj.answer_type.model
 
-    def get_answers(self, obj: Question) -> List:
+    def get_answers(self, obj: Question) -> Any:
```

Comment:
Won't this always be a list? :thinking:

Reply:
probably but it can return Serializer.data which would need some other change to make mypy understand returns a list in this case